internal/services/redhatopenshift/redhat_openshift_cluster_resource.go

```diff
@@ -310,7 +310,7 @@ func (r RedHatOpenShiftCluster) Arguments() map[string]*pluginsdk.Schema {
 			Type:     pluginsdk.TypeList,
 			Required: true,
 			ForceNew: true,
-			MaxItems: 1,
+			MinItems: 1,
```

Comment:
Since we're allowing multiple items now, how is the ordering handled by the API? Is it preserved?

Reply:
First, thanks for the detailed review - I need a bit of time to integrate all the changes due to project load and vacation in the next 2 weeks. From practical use (recreated the cluster about 6 times, using two ingresses, but always with the same names), the sort order seems to be stable and equal to the order in which the ingresses got defined. However, I don't know whether the logic behind this sort order is publicly accessible as source code anywhere.